riot_pal/serial_driver.py

```diff
@@ -48,11 +45,20 @@ def _connect(self, *args, **kwargs):
                     kwargs['port'] = os.environ['PORT']
                 else:
                     kwargs['port'] = self.DEFAULT_PORT
+
+        if 'connect_wait' not in kwargs:
+            if 'CONNECT_WAIT' in os.environ:
+                kwargs['connect_wait'] = os.environ['CONNECT_WAIT']
```

Comment:
mhm, I wouldn't do it that way, i.e. look for an environment variable, however having a default is good.

Also not using env variable you can simplify this to:

```
connect_wait= kwargs.pop('connect_wait', self.DEFAULT_CONNECT_WAIT)
```

which makes the whole `if ... if` obsolete, see also [here](https://docs.python.org/3/library/stdtypes.html#dict.pop) for pop with default

Reply:
Heh makes sense.  That means if we will need to pass the settings in args only.  If that is standard then maybe we do that for everything (I already think we are)?

Reply:
Done.